ddtrace/propagation/_database_monitoring.py

```diff
@@ -95,3 +123,12 @@ def _get_dbm_comment(self, db_span):
             # replace leading whitespace with trailing whitespace
             return sql_comment.strip() + " "
         return ""
+
+    def _find_first_tag(self, db_span, tags):
```

Comment:
nit: since this doesn't use self/cls, you can choose to make it module (instead of class) level